src/main/java/net/minecraftforge/common/ISpecialArmor.java

```diff
@@ -86,6 +86,19 @@
      */
     void damageArmor(EntityLivingBase entity, @Nonnull ItemStack stack, DamageSource source, int damage, int slot);
 
+    /**
+    * Simple check to see if the armor should interact with "Unblockable" damage
+    * sources. A fair number of vanilla damage sources have this tag, such as
+    * Anvils, Falling, Fire, and Magic.
+    *
+    * Returning true here means that the armor is able to meaningfully respond
+    * to this damage source. Otherwise, no interaction is allowed.
+    */
+    default boolean handleUnblockableDamage()
```

Comment:
I think it would be good to pass in the same args as getProperties so mods can decide this based on more things

Reply:
There's not really a need. That can be handled as part of getProperties().

This is simply a check to determine if it should be handled at all. You can have additional handling in getProperties() to determine how to handle various *types* of unblockable damage.

Reply:
I can add the params, but it basically ends up as bloat.

Reply:
Yes that would indeed be a good change, However seeing as we need to return an ArmorProperties with whom we can filter out DamageSources it should not really matter.